packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC } from 'react'
+
+import hero576 from '@/assets/images/ypp-hero/hero-576.webp'
+import hero864 from '@/assets/images/ypp-hero/hero-864.webp'
+import hero1152 from '@/assets/images/ypp-hero/hero-1152.webp'
+import hero2304 from '@/assets/images/ypp-hero/hero-2304.webp'
+import yt576 from '@/assets/images/ypp-hero/yt-576.webp'
+import yt864 from '@/assets/images/ypp-hero/yt-864.webp'
+import yt1152 from '@/assets/images/ypp-hero/yt-1152.webp'
+import yt2304 from '@/assets/images/ypp-hero/yt-2304.webp'
+import { GridItem, LayoutGrid } from '@/components/LayoutGrid'
+import { Text } from '@/components/Text'
+import { Button } from '@/components/_buttons/Button'
+import { SvgActionChevronR } from '@/components/_icons'
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+import useScrollPosition from '@/hooks/useScrollPosition'
+
+import { BackImage, FrontImage, HeroImageWrapper, StyledLimitedWidthContainer } from './YppLandingView.styles'
+
+export const YppLandingView: FC = () => {
+  const mdMatch = useMediaMatch('md')
+  const smMatch = useMediaMatch('sm')
+  const frontImagePosition = useScrollPosition(smMatch ? 0.2 : 0.1)
```

Comment:
This is very inefficient way to do parallax - you are listening to global scroll event and changing state based on that. As a result, when scrolling, this view will be rendered hundreds of times because of all the state changes. To alleviate you would need following changes:
1) Isolating re-renders of parallax images
2) Throttling scroll event
3) Probably a bunch of other things. Take a look at the performance section of the lib you mentioned the other time: https://parallax-controller.damnthat.tv/docs/performance There's a bunch of things they do to optimize.

I think our own JS-based solution will not work well, so let's do one of the following:
a) we do pure CSS based parallax, without using JS to track positions (not sure if fully possible)
b) we use an optimized library that does the heavy lifting for us